runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/MapClue.java

```diff
@@ -30,22 +30,47 @@
 import java.util.Set;
 import lombok.Getter;
 import net.runelite.api.GameObject;
+import static net.runelite.api.ItemID.CLUE_SCROLL_EASY_12179;
+import static net.runelite.api.ItemID.CLUE_SCROLL_EASY_2713;
+import static net.runelite.api.ItemID.CLUE_SCROLL_EASY_2716;
+import static net.runelite.api.ItemID.CLUE_SCROLL_EASY_2719;
+import static net.runelite.api.ItemID.CLUE_SCROLL_EASY_3516;
+import static net.runelite.api.ItemID.CLUE_SCROLL_EASY_3518;
+import static net.runelite.api.ItemID.CLUE_SCROLL_EASY_7236;
+import static net.runelite.api.ItemID.CLUE_SCROLL_ELITE_12130;
+import static net.runelite.api.ItemID.CLUE_SCROLL_HARD;
+import static net.runelite.api.ItemID.CLUE_SCROLL_HARD_3520;
+import static net.runelite.api.ItemID.CLUE_SCROLL_HARD_3522;
+import static net.runelite.api.ItemID.CLUE_SCROLL_HARD_3524;
+import static net.runelite.api.ItemID.CLUE_SCROLL_HARD_3525;
+import static net.runelite.api.ItemID.CLUE_SCROLL_HARD_7239;
+import static net.runelite.api.ItemID.CLUE_SCROLL_HARD_7241;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_3596;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_3598;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_3599;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_3601;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_3602;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_7286;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_7288;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_7290;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_7292;
+import static net.runelite.api.ItemID.CLUE_SCROLL_MEDIUM_7294;
```

Comment:
Not sure if removing the star import is warranted. We don't have all map clues, so this list of imports will be quite long in the end I think.